src/components/TaskModal.vue

```diff
@@ -130,6 +132,13 @@ export default {
 		}
 	},
 	methods: {
+		/**
+		 * @param {string} id The calendar id
+		 * @return {object|undefined} The calendar object (if it exists)
+		 */
+		getCalendarById(id) {
+			return this.calendars.find((cal) => cal.id === id)
+		},
```

Comment:
@st3iny the initial error is gone, but now _Invalid prop: type check failed for prop "displayname". Expected String, got Undefined_  same for color, is shown, i think its because of this. Am i doing this right?